docs/html/development/getting-started.rst

```diff
@@ -38,13 +38,25 @@ To run the pip executable from your source tree during development, install pip
 locally using editable installation (inside a virtualenv).
 You can then invoke your local source tree pip normally.
 
-.. code-block:: console
+.. tabs::
+
+   .. group-tab:: Unix/macOS
+
+      .. code-block:: shell
+
+         $ virtualenv venv # You can also use "python -m venv venv" from python3.3+
+         $ source venv/bin/activate
+         $ python -m pip install -e .
+         $ python -m pip --version
+
+   .. group-tab:: Windows
 
-    $ virtualenv venv # You can also use "python -m venv venv" from python3.3+
-    $ source venv/bin/activate
-    $ python -m pip install -e .
-    $ python -m pip --version
+      .. code-block:: shell
 
+         C:\> virtualenv venv # You can also use "py -m venv venv" from python3.3+
+         C:\> source venv/bin/activate
```

Comment:
Does `source` work on Windows?

https://stackoverflow.com/q/8921188/724176

Reply:
Thank you. it should be `venv\Scripts\activate` instead. Will push this change shortly.